### ISSUE TITLE
Python 3 support?

### ISSUE THREAD

**Issue by briankung** [zapier/django-rest-hooks#19]
The tests seem to be passing just fine with Python 3 and Django 1.7.7. I'm not sure if this means that it is compatible with Python3, however. Is it python3 compatible?

**Comment by avelis** [comment]
@briankung To answer frankly, not officially.

If you feel inclined to offer a pull request with Python3 support I can gladly take a look and merge it in.

This the first official request for Python3 support and I am not against the consideration.

**Comment by briankung** [comment]
Thanks for the quick reply @avelis. I likely won't be able to submit a pull request in the near future, unfortunately, but thank you for clarifying :+1:

**Comment by avelis** [comment]
@briankung My previous statement is false.
Upon checking our travis yml file: https://github.com/zapier/django-rest-hooks/blob/master/.travis.yml

We run unit tests against Python 3.3 & Python 3.4.

Are those versions of Python sufficient for you?

**Comment by briankung** [comment]
That's awesome, and yes, those versions should be sufficient. I'm currently using 3.4.3. I'm glad `django-rest-hooks` is an option for us. It is by far the most popular library on djangopackages.com for webhooks and I trust Zapier to know what they're doing with webhooks.

By the way, I ran [modernize](https://pypi.python.org/pypi/modernize) on the repo and it didn't have any major changes: https://gist.github.com/04f7d9f70cb61df377c4. The only manual addition I made was to insert `six` in `devrequirements.txt` so that it wouldn't break under python 3+.

I'm sure you know better than I, but my guess is that there are no major compatibility issues and modernize's suggestions can largely be ignored, especially seeing as you have travis configurations for the major python versions.

Thanks for investigating!

PS - it would be nice if the python 3 compatibility was updated on Django Packages: https://www.djangopackages.com/packages/p/django-rest-hooks/  I'm not sure how to mark it as compatible, or I would. Thanks again :+1:
